src/collections/_documentation/development/server/environment.md

```diff
@@ -1,57 +1,38 @@
 ---
 title: 'Development Environment'
-sidebar_order: 1
+sidebar_order: 2
 ---
 
-## Setting up Your Development Environment
+This guide steps you through configuring a local development environment for the Sentry server on macOS. If you're using another operating system the instructions are still roughly the same, but we don't maintain any official documention.
 
-### Linux
+#### A quick note on Bash vs Zsh...
 
-Follow [the Docker file in the repository](https://github.com/getsentry/sentry/blob/master/docker/Dockerfile) but instead of creating and installing a wheel, install the package in editable mode (`pip install . -e`).
+macOS Catalina will recommend that you use `zsh` instead of `bash`. While the differences are minimal, this guide primarily covers Bash. Verify your shell by running `echo $SHELL`.
 
-### macOS
+## Clone the Repository
 
-{% capture __alert_content -%}
-macOS Catalina will recommend that you use the zsh shell, instead of bash. While the differences are minimal, this guide only covers bash, so for best results please use bash. You can verify this by typing `echo $SHELL`, which should result in `/bin/bash`.
-
-{%- endcapture -%}{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-  level="warning"
-%}
-
-To get started, fork the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry). Clone your forked repo and go into it:
+To get started, clone the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry) - or, your fork if you are not a member of `@getsentry`.
 
 ```bash
-git clone https://github.com/<your github username>/sentry.git
+git clone https://github.com/getsentry/sentry.git
 cd sentry
 ```
```

Comment:
I figured we'd take a pass at the second round of changes which talk about sentry vs getsentry - I'm fine with explaining that, or even having a second doc that embeds part of this one for staff, At the very least no reason it can't be public, and we can just ensure that both setups mimic each other.